src/common/copilot/assets/scripts/copilot.js

```diff
@@ -79,7 +80,7 @@
         return resultDiv;
       }
 
-      if (responseText[i].code === "" || responseText[i].code === null || responseText[i].code === undefined || responseText[i].code === "violation" || responseText[i].code === "unclear") {
+      if (responseText[i].code === "" || responseText[i].code === null || responseText[i].code === undefined || responseText[i].code === "violation" || responseText[i].code === "unclear" || responseText[i].code === "explain") {
```

Comment:
we can consider converting this into enum - instead of using hard coded string here.